LayoutTests/imported/w3c/web-platform-tests/html/links/301-response.py

```diff
@@ -0,0 +1,27 @@
+def main(request, response):
+    response.status = 200
```

Comment:
Why is this script named 301-response.py given it returns a 200 response?
It might be good to add tests for 301 and others redirection HTTP codes.

It might also be good to add tests for redirection URLS with location URLs having frag IDs if we do not cover this case.